content/hi/docs/setup/best-practices/_index.md

```diff
@@ -0,0 +1,4 @@
+---
+title: सर्वोत्तम प्रथाएं
```

Comment:
@anubha-v-ardhan Can we go with more simple wording, like https://github.com/kubernetes/website/issues/29353#issuecomment-898821901

Reply:
According to me `सर्वोत्तम प्रथाएं` is easy and clear Hindi. `सर्वोत्तम प्रथाएं` is not a core Kubernetes terminology so we can have it in pure Hindi. If you have any simpler Hindi alternative for `सर्वोत्तम प्रथाएं`, please recommend. I can't think of any alternative with same clarity and meaning. : )

Reply:
How about this `बेस्ट प्रैक्टिसेज`?

Reply:
`best practices` is not a technical term, I can't see any reason to transliterate it. Should I change it to `बेस्ट प्रैक्टिसेज`?

Reply:
Hmm... In that case, can you look for more common word. `सर्वोत्तम` is not a word we use in regular conversation.

Reply:
Its not used in regular conversation but it is pretty much understandable by any Hindi speaker. Daily use words like "sabse aacha" will sound very informal

Reply:
The sense you want to convey is similar to “methods that are commonly recognised as effective”. उत्तम प्रचलन feels like a good phrase, based on how I'm seeing it used on other web pages.

Reply:
Did folks decide what wording to adopt?

Reply:
I believe we can go with उत्तम प्रचलन

cc @mittalyashu